src/components/Nav/MyPageLink.js

```diff
@@ -1,32 +1,53 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router-dom';
 import './Nav.scss';
 
 class myPageLink extends Component {
+  logOut = () => {
+    localStorage.removeItem('TOKKEN');
+    this.props.history.push('/');
+  };
   render() {
+    const token = localStorage.getItem('TOKKEN');
     return (
       <div className="myPageLink">
         <ul>
-          {MY_PAGE_LINK.map(el => {
-            const { key, content } = el;
-            return (
-              <li key={key}>
-                <Link to="/">{content}</Link>
-              </li>
-            );
-          })}
+          {token
+            ? NAV_LINK_TOKENIN.map(menu => {
+                const { key, content, url } = menu;
+                return (
+                  <li key={key}>
+                    <Link to={url}>{content}</Link>
+                  </li>
+                );
+              })
+            : NAV_LINK_TOKENOUT.map(menu => {
+                const { key, content, url } = menu;
+                return (
+                  <li key={key}>
+                    <Link to={url}>{content}</Link>
+                  </li>
+                );
+              })}
```

Comment:
```suggestion
          {(token ? NAV_LINK_TOKENIN : NAV_LINK_TOKENOUT).map(menu => {
                const { key, content, url } = menu;
                return (
                  <li key={key}>
                    <Link to={url}>{content}</Link>
                  </li>
                );
              })
```
전달되는 배열만 바뀌고 내용은 동일하니까 이와 같이 중복 줄여볼 수 있을거 같아요

Reply:
왕,,,,, 수정완료했습니다!!!